File: EjerciciosC++/fibonacciConejos/fibonacciConejos/fibonacci.cpp
#include <iostream>
using namespace std;

int main()
{

	int conejosTiernos, conejosAdultos, meses, aux;

	cout << "Introuduzca de meses que desea evaluar: ";
	cin >> meses;

	cout << endl << "Introduzca la cantidad de conejos tiernos con los que desea empezar: ";
	cin >> conejosTiernos;

	cout << endl << "Introduzca la cantidad de conejos adultos con los que desea empezar: ";
	cin >> conejosAdultos;

	for (int i = 1; i <= meses; i++) {

		aux = conejosAdultos;
		conejosAdultos += conejosTiernos;
		conejosTiernos = aux;

	}

	cout << "La cantidad de conejos en " << meses << " meses es "
		<< conejosTiernos + conejosAdultos << endl;

	system("PAUSE");
	return 0;
}

File: Ejercicios C#/fibonacci/fibonacci/fibonacci.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace fibonacci
{
    class fibonacci
    {
        static void Main()
        {

            int conejosTiernos, conejosAdultos, meses, mes, auxiliar;

            Console.Write("Introduzca el numero de conejos tiernos con los que desea empezar: ");
            conejosTiernos = int.Parse(Console.ReadLine());

            Console.Write("Introduzca el numero de conejos adultos con los que desea empezar: ");
            conejosAdultos = int.Parse(Console.ReadLine());

            Console.Write("Introduzca el mes que desea ver: ");
            meses = int.Parse(Console.ReadLine());

            for (mes = 1; mes <= meses; mes++)
            {

                auxiliar = conejosAdultos;
                conejosAdultos += conejosTiernos;
                conejosTiernos = auxiliar;

            }

            Console.WriteLine("El numero de conejos en el mes {0} es {1}", meses, conejosTiernos + conejosAdultos);

            Console.Write("Presione una tecla para continuar ...");
            Console.Read();

        }
    }
}
